src/test/java/com/wikia/webdriver/testcases/specialpagestests/EditingPreferencesTests.java

```diff
@@ -65,15 +61,36 @@ public void EditPreferences_002_selectCK() {
   }
 
   @Test(groups = {"EditingPreferencesTest", "EditPreferences_003"})
+  @Execute(asUser = User.USER_5, onWikia = URLsContent.VE_ENABLED_WIKI)
   public void EditPreferences_003_selectSource() {
-    EditingPreferencesPageObject editPrefPage = base.openSpecialEditingPreferencesPage(wikiURL);
+    EditPreferencesPage editPrefPage = new EditPreferencesPage(driver).openEditingSection();
     editPrefPage.selectPreferredEditor(SOURCE);
     PreferencesPageObject prefPage = editPrefPage.clickSaveButton();
     prefPage.verifyNotificationMessage();
-    String articleName = PageContent.ARTICLE_NAME_PREFIX + base.getTimeStamp();
+    String articleName = PageContent.ARTICLE_NAME_PREFIX + DateTime.now().getMillis();
     ArticlePageObject aritclePage = new ArticlePageObject(driver).open(articleName);
     SourceEditModePageObject src = aritclePage.openSrcModeWithMainEditButton();
     src.verifySourceOnlyMode();
   }
-}
 
+  @Test(groups = {"EditingPreferencesTest", "EditPreferences_003"})
+  @Execute(asUser = User.USER_5)
+  public void changeEmailAddress() {
+    final String newEmailAddress = "myAwesomeEmail@email.co.uk";
```

Comment:
This is a local final field, It's here so no other guy can modify it after it's initialized. Moving it to class field is a horrible idea.